src/main/java/linus/Linus.java

```diff
@@ -0,0 +1,155 @@
+package linus;
+
+import linus.exception.LinusException;
+import linus.task.Deadline;
+import linus.task.Event;
+import linus.task.TaskList;
+import linus.task.ToDo;
+import linus.util.Parser;
+import linus.storage.Storage;
+import linus.util.Ui;
+
+/**
+ * Represents a Linus Chatbot.
+ */
+public class Linus {
+    private static final String FILE_PATH = "data/linus.txt";
+
+    private static enum Command {
+        LIST,
+        MARK,
+        UNMARK,
+        DELETE,
+        FIND,
+        TODO,
+        DEADLINE,
+        EVENT,
+        HELP,
+        BYE
+    }
+
+    private Storage storage = null;
+    private TaskList tasks = null;
+
+    private Ui ui = null;
+
+    /**
+     * Constructs a Linus Chatbot with the specified file path.
+     *
+     * @param filePath The file path.
+     */
+    public Linus(String filePath) {
+        storage = new Storage("data/linus.txt");
```

Comment:
As you have a constant FILE_PATH above, perhaps it's a good idea to use it here?